v1/mutate/mutate.go

```diff
@@ -132,6 +132,10 @@ func Config(base v1.Image, cfg v1.Config) (v1.Image, error) {
 		diffIDMap:  make(map[v1.Hash]v1.Layer),
 		digestMap:  make(map[v1.Hash]v1.Layer),
 	}
+	image.manifest.Config.Digest, err = image.ConfigName()
```

Comment:
I haven't read through all of this file, but we do this above in `Append`, do we need to do it here too?

Reply:
This doesn't call append :)